<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogJin</class>
 <widget class="QDialog" name="DialogJin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>695</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonCalculate">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>460</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>611</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAll">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款总额</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelRate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款月利率</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTime">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>20</y>
     <width>81</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>贷款期限</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAll">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditTime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditRate">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>460</y>
     <width>71</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeYear">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>60</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>年</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTimeWan">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>万</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAllShoufu">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>460</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>首付金额（万）：</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelAllBank">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>银行利率（年）：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAShoufu">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAllBank">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGong">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>每月公积金</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditGong">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>60</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
